eventmesh-common/src/main/java/org/apache/eventmesh/common/config/ConfigService.java

```diff
@@ -0,0 +1,173 @@
+     */
+    private Properties properties = new Properties();
+
+    @Getter
+    private String rootPath;
+
+    private final ConfigMonitorService configMonitorService = new ConfigMonitorService();
+
+    private String configPath;
+
+
+    public static ConfigService getInstance() {
+        return INSTANCE;
+    }
+
+    public ConfigService() {
+    }
+
+    public ConfigService setConfigPath(String configPath) {
+        this.configPath = configPath;
+        return this;
+    }
+
+    public void setRootConfig(String path) throws Exception {
+        ConfigInfo configInfo = new ConfigInfo();
+        rootPath = path;
+        configInfo.setPath(rootPath);
+        properties = this.getConfig(configInfo);
+    }
+
+    public Properties getRootConfig() {
+        return this.properties;
+    }
+
+    public <T> T buildConfigInstance(Class<?> clazz) {
+        Config[] configArray = clazz.getAnnotationsByType(Config.class);
+        if (configArray.length == 0) {
+            try {
+                return this.getConfig(ConfigInfo.builder()
+                        .clazz(clazz)
+                        .hump(ConfigInfo.HUMP_SPOT)
+                        .build());
+            } catch (Exception e) {
+                throw new RuntimeException(e);
+            }
+        }
+
+        Config config = configArray[0];
+        try {
+            // todo Complete all attributes
+            ConfigInfo configInfo = new ConfigInfo();
+            configInfo.setClazz(clazz);
+            configInfo.setPath(config.path());
+            configInfo.setHump(config.hump());
+            configInfo.setPrefix(config.prefix());
+            configInfo.setMonitor(config.monitor());
+
+            return this.getConfig(configInfo);
+        } catch (Exception e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void populateConfigForObject(Object object) throws IllegalAccessException, NoSuchFieldException, IOException {
+        Class<?> clazz = object.getClass();
+        Config[] configArray = clazz.getAnnotationsByType(Config.class);
+        if (configArray.length == 0) {
+            return;
+        }
+
+        for (Config config : configArray) {
+            populateConfig(object, clazz, config);
+        }
+    }
+
+    @SuppressWarnings("unchecked")
+    public <T> T getConfig(ConfigInfo configInfo) throws IOException {
+        Object object;
+
+        if (Objects.isNull(configInfo.getPath()) || StringUtils.isEmpty(configInfo.getPath().trim())) {
+            object = FileLoad.getPropertiesFileLoad().getConfig(properties, configInfo);
+        } else {
+            String path = configInfo.getPath();
+            String filePath;
+            if (path.startsWith("classPath://")) {
+                filePath = Objects.requireNonNull(ConfigService.class.getResource("/" + path.substring(12))).getPath();
+            } else if (path.startsWith("file://")) {
+                filePath = path.substring(7);
```

Comment:
I think length("file://")  to replace 7 is better.

Reply:
Done!